test/base-tlsutility.cpp

```diff
@@ -132,4 +150,165 @@ BOOST_AUTO_TEST_CASE(iscertuptodate_old)
 	})));
 }
 
+// openssl req -x509 -newkey ec -pkeyopt ec_paramgen_curve:secp384r1 -subj /CN=IcingaCA -days 1000000 -keyout IcingaCA.key -out IcingaCA.crt -nodes
```

Comment:
This looks a bit different than our usual RSA certificates. Maybe this is not so representative?

Reply:
Fair enough. Switched back to RSA.

Reply:
I'll switch to our usual RSA x4096, now as the certs are in a dedicated file...